api/v1beta1/nutanixcluster_validate_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2024 Nutanix
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1_test
```

Comment:
just wondering how did it allow 2 package names in one dir. v1beta1 and v1beta1_test

Reply:
it's not an issue of allow vs not allow: golang has no issues with that. I'm more curious about the choice of using a separate `pkg_test` package vs just the `pkg` itself.

Reply:
Because there was no actual code under test here, it seemed to make sense to me.